plugins/compress/configuration.cc

```diff
@@ -190,6 +195,15 @@ HostConfiguration::is_status_code_compressible(const TSHttpStatus status_code) c
   return it != compressible_status_codes_.end();
 }
 
+std::string_view
+strip_params(std::string_view v)
+{
+  swoc::TextView tv{v};
+  tv = tv.take_prefix_at(';');
+  tv.rtrim_if([](int ch) { return std::isspace(ch); });
```

Comment:
Actually, I wonder if this will work?

`tv.rtrim_if(&isspace);`

Reply:
and we can combine the two lines as well?

`tv = tv.take_prefix_at(';').rtrim_if(&isspace);`

Reply:
> Actually, I wonder if this will work?
> 
> `tv.rtrim_if(&isspace);`

This does not compile unfortunately